kits/cpp/simple/lux/map.hpp

```diff
@@ -24,7 +24,7 @@ namespace lux
         Position pos = Position(-1, -1);
         Resource resource;
         lux::CityTile * citytile;
-        float cooldown = 1.0;
+        float road = 0.0;
```

Comment:
good catch

Reply:
Yeah, I caught that bug but forgot to mention it :-D